documentation/service/technical/searching-by-location.md

```diff
@@ -0,0 +1,175 @@
+The coordinates are stored as latitude and longitude on the Earth's surface, following the WGS84 standard.
+
+Using the `geometry` type with `geographic: true` allows us to leverage a wide range of PostGIS geospatial functions, such as `ST_Distance`, `ST_Within`, `ST_Intersects`, and `ST_Buffer`.
+
+### Meaning of `SRID 4326`
+
+- SRID: Spatial Referencing System ID.
+- 4326: Represents spatial data using longitude and latitude coordinates on the Earth's surface as defined in the WGS84 standard.
+
+The data type we use from PostGIS is `Geometry` with a `Geographic` coordinate system, which allows us to use projections and transformations.
+
+### Why not just Geometry
+
+When using a geographic coordinate system, distance calculations are performed on the curved surface of the Earth.
+PostGIS functions like `ST_Distance` and `ST_Buffer` will take into account the Earth's curvature, providing more accurate results for large distances compared to a planar (flat) coordinate system.
+
+When investigating the possibility of using Geometry and a Projected Coordinate System, we found that our search area (England, The UK) distances are large enough to need to take the Earth's curvature into account.
+
+### Performance
+
+Using `Geographic` data and operations requires more computational resources than using planar `Geometric` data.
+
+We have mitigated the performance hit with a few measures:
+
+#### Spatial indexing
+
+We use PostGIS spatial indexing for any geospatial data used in our queries. Stored areas and points must have spatial indexes when used in any querying.
+
+#### Reducing Location Polygon areas complexity
+
+We have experienced major performance issues when some of our location polygons coming from ONS have tens of thousands of points (e.g., the Cornwall polygon consisted of 125k points).
+
+Doing an `ST_Buffer` (expanding the polygon for radius searches) over polygons with that level of complexity is very taxing on the database CPU and memory.
+
+Simplifying the polygons with some tolerance offers a major reduction in the polygon points while quite accurately keeping the polygon shape.
+For example, the Cornwall polygon with 2.5k points instead of 125k is almost identical to the original while being way less resource-expensive to operate with.
+
+To achieve that, we use `ST_SimplifyPreserveTopology` over the ONS imported polygons prior to storing them in our database.
+
+#### Precomputing data when possible
+
+Precomputing the `centroid` point for the location polygon areas and saving them in the database improves our search by distance queries performance. It avoids calculating each location polygon center to use it to calculate the polygon distance from each vacancy. Instead, it uses the stored centroid to calculate the distance.
+
+#### Tweaking queries to improve performance
+
+The `use_spheroid` parameter set to `false` turns to use a faster spherical calculation.
```

Comment:
[Minor] This sentence is a bit confusing imo. Is 'turns' a technical term for something here or maybe a typo?